crates/cli-support/src/wit/standard.rs

```diff
@@ -308,6 +309,8 @@ pub enum Instruction {
     OptionEnumFromI32 {
         hole: u32,
     },
+    InNonNullSentinel,
+    OutNonNullSentinel,
```

Comment:
I'm not particularly a fan of these names; all the other `Instruction`s tend to use `From` and `To`, not `In` and `Out`, so they don't match very well. `OptionNonNullFromI32` and `I32FromOptionNonNull` would be more in line with everything else.